libraries/Microsoft.Bot.Schema/Context/RecognizerDescription.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Schema
+{
+    /// <summary>
+    /// Description of the intents and entities a recognizer can return.
+    /// </summary>
+    public partial class RecognizerDescription
```

Comment:
>RecognizerDescription [](http://example.com/codeflow?start=25&length=21)

Not a fan of "Description" name...RecognitionReference? Does it require it to be broken down into intents and entities? Isn't the name + source enough for it to know that?  That would simplify and also not bake in LU concepts of intents and entities that may change in the fuiture.

Reply:
We have intents and entities everywhere in the SDK, recognizers, composer, ...  You want to introduce a new concept?  Why is Reference better than description?  You would have a single list of Recognitions?